assets/base.css

```diff
@@ -2266,6 +2276,7 @@ product-info .loading-overlay:not(.hidden) ~ *,
   padding: 1rem 0;
   margin: 0;
   letter-spacing: 0.1rem;
+  min-height: 3.8rem;
```

Comment:
If you're ok with this, I probably am, but historically we do make a best effort to keep things 44px. Just want to double check we don't want to add in the additional 6px. In theory a tablet touch device could display at the desktop breakpoint and still benefit from the 44px min.

Reply:
Yeah, I'm ok with 38px for desktop. 👍